### ISSUE TITLE
docformat sometimes produces output which it changes further on rerunning

### ISSUE THREAD

**Issue by DRMacIver** [myint/docformatter#16]
Consider the following code:

```python
class HealthCheck():
    """Classes of health check
    for use in :attr:`~hypothesis.settings.suppress_health_check`

    """
```

When run on this code, docformat produces the following:

```python
class HealthCheck():
    """Classes of health check for use in
    :attr:`~hypothesis.settings.suppress_health_check`"""
```

Running docformat on this again produces the following:

```python
class HealthCheck():
    """Classes of health check for use in.

    :attr:`~hypothesis.settings.suppress_health_check`

    """
```

Generally I would expect formatting tools to format to fixed points - i.e. once you've run them once, running them again should produce zero changes. Is it intentional that this is not the case here?

**Comment by myint** [comment]
Thanks for the report! This is a bug. The first change you note is a line wrapping. When run again, a [heuristic](https://github.com/myint/docformatter/blob/c2b87f2d068a20ed5d9f68cd4b7cd07c13f9705a/docformatter.py#L261) takes affect that didn't get triggered in the previous case. A temporary workaround would be to make `is_probably_beginning_of_sentence()` less zealous. But there may be a better approach to this.

**Comment by myint** [comment]
Closed by #41.
